File: src/test/java/org/zbus/examples/mq/ProducerExample.java
package org.zbus.examples.mq;

import org.zbus.broker.Broker;
import org.zbus.broker.SingleBroker;
import org.zbus.mq.Producer;
import org.zbus.net.http.Message;

public class ProducerExample {
	public static void main(String[] args) throws Exception {
		Broker broker = new SingleBroker(); // default to 127.0.0.1:15555

		Producer producer = new Producer(broker, "MyMQ"); 
		producer.createMQ();

		Message msg = new Message(); 
		msg.setBody("hello world"); 
		msg = producer.sendSync(msg);
		System.out.println(msg);

		broker.close();
	}
}

File: clients/DotNET/Zbus.Client.Examples/Mq/ProducerExample.cs
using Api.Example;
using Newtonsoft.Json;
using System;
using System.Threading;
using System.Threading.Tasks;
using Zbus.Client.Broker;
using Zbus.Client.Mq;
using Zbus.Client.Net.Http;
using Zbus.Client.Rpc;

namespace Zbus.Client.Examples
{
   class ProducerExample
   {  
      static void Main(string[] args)
      {
         using (IBroker broker = new ZbusBroker("127.0.0.1:15555"))
         {
            Producer p = new Producer(broker, "MyMQ");
            Message msg = new Message
            {
               BodyString = "hello world"
            };
            Message res = p.Send(msg); ;

            Console.WriteLine(res); 
         }

         Console.ReadKey();
      }
   }
}
